content/how-to/add-code-servers/bitbucket-cloud/_index.md

```diff
@@ -37,7 +37,7 @@ The next step in adding the code server is adding the Personal Access Token (PAT
 
 Example: `holland:Vgc9Kw8w5zQu58989Uxz`
 
-[how to generate a personal access token on Bitbucket](https://support.blubracket.com/hc/en-us/articles/4404687343124-How-to-Generate-an-App-Password-or-Personal-Access-Token-PAT-in-Bitbucket)
+[How to generate a personal access token on Bitbucket](/how-to/add-code-servers/bitbucket-cloud/access-token/)
```

Comment:
Should the path for this be consistent with the path for the GitLab PAT instructions below? Maybe the consistent and short version is `generate-pat`?

Reply:
Yes this should be consistent I'm going to change it to generate-pat for both.